### PR TITLE
Create passwordless sudoer `crc_ci` for CentosCI

### PR DESCRIPTION
**Pull request by jsliacan**
To avoid running tests under `root`, create a `crc_ci` user with passwordless sudo privileges.

### CODE REVIEW

**Comment by anjannath** [comment]
We can remove the previous unzip in: https://github.com/code-ready/crc/pull/301/files#diff-15c128a70ef5dd7aaf264bce566298fdL91

File: centos_ci.sh [hunk 1]
```diff
@@ -88,7 +110,6 @@ function perform_artifacts_upload() {
 function get_bundle() {
   mkdir $HOME/Downloads
   curl -L 'https://github.com/code-ready/crc-ci-jobs/releases/download/4.0.1/crc_bundle.zip' -o $HOME/Downloads/bundle.zip
```
Comment (gbraad) [inline centos_ci.sh]: Do we use this? If not, this is conflicting with what is in https://github.com/code-ready/crc/pull/301/files#diff-15c128a70ef5dd7aaf264bce566298fdR128 ?

Comment (jsliacan) [reply centos_ci.sh]: We download `bundle.zip` to `/root/Downloads`, then copy it to `home/crc_ci/Downloads` after `crc_ci` is created, then unzip it there - this results in `/home/crc_ci/crc_libvirt_4.1.3.tar.xz`.

File: centos_ci.sh [hunk 2]
```diff
@@ -104,28 +120,36 @@ function upload_logs() {
 
 function run_tests() {
   set +e
-  make integration BUNDLE_LOCATION="/root/Downloads/crc_libvirt_4.1.3.tar.xz"
+  make integration BUNDLE_LOCATION=$HOME/Downloads/crc_libvirt_4.1.3.tar.xz
```
Comment (gbraad) [inline centos_ci.sh]: would prefer if we can extract this as a variable (global)

File: centos_ci.sh [hunk 3]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# bundle location
+BUNDLE=$HOME/Downloads/crc_libvirt_4.1.3.tar.xz
```
Comment (gbraad) [inline centos_ci.sh]: maybe JUST the file name?, ome and downloads is defined by the extract location

File: centos_ci.sh [hunk 4]
```diff
@@ -104,28 +123,36 @@ function upload_logs() {
 
 function run_tests() {
   set +e
-  make integration BUNDLE_LOCATION="/root/Downloads/crc_libvirt_4.1.3.tar.xz"
+  make integration BUNDLE_LOCATION=$BUNDLE
```
Comment (gbraad) [inline centos_ci.sh]: `/home/.../Downloads/$BUNDLEFILENAME`

**Pull request merged**